src/PrestaShopBundle/Entity/Repository/StockManagementRepository.php

```diff
@@ -380,4 +396,131 @@ protected function bindMaxResultsValue(Statement $statement, QueryParamsCollecti
             PDO::PARAM_INT
         );
     }
+
+    /**
+     * Store the number of rows found in a previous query executed with SQL_CALC_FOUND_ROWS
+     */
+    protected function getFoundRows()
+    {
+        $statement = $this->connection->prepare('SELECT FOUND_ROWS()');
+        $statement->execute();
+        $rowCount = (int)$statement->fetchColumn();
+        $statement->closeCursor();
+
+        return $rowCount;
+    }
+
+    /**
+     * @param array $row
+     */
+    protected function addCombinationName(array &$row)
```

Comment:
how about rename it to getCombinationName and remove the need of passing an array by reference but the real required information?